sdk/openai/openai/test/public/browser/webSocketRealtime.spec.ts

```diff
@@ -0,0 +1,76 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { matrix } from "@azure-tools/test-utils-vitest";
+import { describe, beforeEach, it } from "vitest";
+import { APIVersion, withDeployments } from "../../utils/utils.js";
+import { OpenAIRealtimeWebSocket } from "openai/beta/realtime/websocket";
+import { createClientsAndDeployments } from "../../utils/createClients.js";
+import {
+  assertResponseTextDeltaEvent,
+  assertResponseTextDoneEvent,
+  assertSessionCreatedEvent,
+} from "../../utils/asserts.js";
+import type { ClientsAndDeploymentsInfo } from "../../utils/types.js";
+
+describe("Realtime", () => {
+  matrix([[APIVersion.Realtime]] as const, async function (apiVersion: APIVersion) {
+    describe(`[${apiVersion}] Client`, () => {
+      let clientAndDeployments: ClientsAndDeploymentsInfo;
+
+      beforeEach(async function () {
+        clientAndDeployments = createClientsAndDeployments(apiVersion, { realtime: "true" });
+      });
+
+      describe("OpenAIRealtimeWebSocket", function () {
+        it("websocket.azure", async function () {
+          await withDeployments(clientAndDeployments, async (client, deploymentName) => {
+            const rt = await OpenAIRealtimeWebSocket.azure(client as any, { deploymentName });
```

Comment:
This is no longer needed after bumping the package version:
```suggestion
            const rt = await OpenAIRealtimeWebSocket.azure(client, { deploymentName });
```